windows/hid.c

```diff
@@ -1003,6 +1030,16 @@ HID_API_EXPORT hid_device * HID_API_CALL hid_open_path(const char *path)
 	return dev;
 }
 
+static int hid_write_output_report(hid_device *dev, const unsigned char *data, size_t length)
+{
+	BOOL res;
+	res = HidD_SetOutputReport(dev->device_handle, (void *)data, (ULONG)length);
```

Comment:
please see https://github.com/libusb/hidapi/pull/224
I'm confident we need to do the same thing for `HidD_SetOutputReport` as well.

Reply:
@slouken any thoughs/comments?

Reply:
@slouken It seems, you missed to respond to this question

Reply:
It looks like #224 was just pre-allocating the output buffer. Were you referring to another change that made sure the buffer size matched feature_report_length? I think, but I'm not sure, that the feature_report_length is the wrong size in this case.

Reply:
I mean that the buffer size has to be at least `output_report_length` bytes in size.
If user passes less data - logically it is not a proble, but WinAPI implementation is known to crash if bufer is too small.
The workaround (as in #224) is to pre-allocate buffer that is large-enough _if needed_.